tests/integration/test_app.py

```diff
@@ -2,55 +2,52 @@
 import db_items as mongoDB
 import os
 import pymongo
-from unittest import mock
-from unittest.mock import Mock
+import mongomock
 from app import create_app
 from dotenv import load_dotenv, find_dotenv
+from unittest.mock import patch
 
-# FILE NEEDS TO BE CHANGED AFTER LEARNING ABOUT MONGOMOCK
+test_todos = [
+    {
+        "board_id": "board1",
+        "list_name": "todolist",
+        "cards": {
+                "card_id": "card1",
+                "card_name": "Shopping",
+                "card_desc": "Groceries to do",
+                "card_dateLastActivity": "2020-08-19T14:09:28.403Z"
+            }
+    },
+    {
+        "board_id": "board1",
+        "list_name": "todolist",
+        "cards": {
+                "card_id": "card2",
+                "card_name": "Music",
+                "card_desc": "Singing to do",
+                "card_dateLastActivity": "2020-07-19T14:09:28.403Z"
+            }
+    }
+]
 
 @pytest.fixture
 def client():
     # Use our test integration config instead of the 'real' version 
-    file_path = find_dotenv('.env.test')
-    load_dotenv(file_path, override=True)
-    # Create the new app.
-    test_app, collection = create_app()
-    # Use the app to create a test_client that can be used in our tests.
-    with test_app.test_client() as client: 
-        yield client
+    with mongomock.patch(servers=(('server.example.com', 27017),)):
+        file_path = find_dotenv('.env.test')
+        load_dotenv(file_path, override=True)
+        # Create the new app.
+        test_app, collection = create_app()
+        # Use the app to create a test_client that can be used in our tests.
+        with test_app.test_client() as client: 
+            yield client
 
-@mock.patch('requests.request')
-def test_index_page(mock_get_requests, client):
-    # Replace call to requests.get(url) with our own function
-    mock_get_requests.side_effect = mock_request
+@mongomock.patch(servers=(('server.example.com', 27017),))
```

Comment:
We don't explicitly need this patch, because the one in the client fixture should still be in play